src/main/java/org/testeditor/fixture/core/DefaultLoggingListener.java

```diff
@@ -0,0 +1,91 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2016 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.fixture.core;
+
+import java.util.concurrent.TimeUnit;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testeditor.fixture.core.TestRunReporter.Position;
+import org.testeditor.fixture.core.TestRunReporter.SemanticUnit;
+
+/**
+ * Default implementation of a test run logger.
+ * Logs enter and leave of TEST.
+ * Logs enter (only) for SPECIFICATION, COMPONENT and STEP
+ */
+public class DefaultLoggingListener implements TestRunListener {

```

Comment:
Let's discuss this ... I'm not sure either. Looked like a nice example for a listener ...